src/Backoffice/Tasks/Domain/DataTransferObjects/TaskDto.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Tasks\Domain\DataTransferObjects;
+
+class TaskDto
+{
+    public function __construct(
+        private readonly string $title,
+        private readonly string $description,
+        private readonly string $status,
```

Comment:
Here use the enum we defined before:

```suggestion
        private readonly TaskStatusEnum $status,
```